Features3/tests/features/addContact.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Add new contact
+    As a user
+    I want to add contact 
+    So that it will be displayed in contact list
+
+    Background:
+        Given User has logged in and navigated to Contact Fox page
+
+    Scenario Outline: Add contact with name, email, contact type and with or without phone number
+        When User enters name as "<name>", email as "<email>", phone as "<phone>" and contact type as "<contact-type>"
+        Then contact with name as "<name>", email as "<email>", phone as "<phone>" and contact type as "<contact-type>" should be created 
+        And contact with name as "<name>", email as "<email>", phone as "<phone>" and contact type as "<contact-type>" should be displayed in contact list
+        Examples:
+            |name|email|phone|contact-type|
+            |abc|abc@gmail.com|9801234567|personal|
+            |xyz|xyz@gmail.com|9874561230|professional|
+            |abc|abc@gmail.com||personal|
+            |xyz|xyz@gmail.com||professional|
+
+    Scenario Outline: Add contact without full details
+        When User enters name as "<name>", email as "<email>" and contact type as "<contact-type>"
+        Then contact with name as "<name>", email as "<email>" and contact type as "<contact-type>" should not be created 
+        Examples:
+            |name|email|contact-type|
+            ||abc@gmail.com|personal|
+            ||abc@gmail.com|professional|
+            |abc||personal|
+            |abc||professional|
+            |||personal|
+            |||professional|
+
+    Scenario: Adding contact with incorrect email format
+        When User adds contact with following details
+            |name|email|phone|contact-type|
+            |abc|abcgmail.com|9801234567|personal|
+            |abc|abcgmail.com|9801234567|professional|
+        Then validation error message as "Please include an '@' in the email address." should pop up in the input field
```

Comment:
Try out multiple combinations, like invalid email, name, phone, etc. Let's not repeat a single case twice.
````
|name|email|phone|contact-type|
|abc|abcgmail.com|9801234567|personal|
|abc|abcgmail.com|9801234567|professional|

Here, both of these cover a single case of invalid email. So no need to have two data, single one would be good enough.
```